### PR TITLE
Drools 3252 - Minor fixes

### PR DESCRIPTION
**Pull request by gitgabrio**
@kkufova @Rikkola @danielezonca 

This PR fix the following tickets:

https://issues.jboss.org/browse/DROOLS-3190
https://issues.jboss.org/browse/DROOLS-3213

To be merged after 
https://github.com/kiegroup/drools-wb/pull/984

### CODE REVIEW

**Comment by kiereleaseuser3** [comment]
Can one of the admins verify this PR? Comment with 'ok to test' to start the build.

**Comment by Rikkola** [comment]
ok to test
I see this is still needed :)

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/test/java/org/drools/workbench/screens/scenariosimulation/client/handlers/ScenarioSimulationGridPanelClickHandlerTest.java [hunk 1]
```diff
@@ -130,56 +130,56 @@ public void setScenarioGrid() {
...
+        scenarioSimulationGridPanelClickHandler.setHeaderGivenContextMenu(headerGivenContextMenuMock);
+        assertEquals(headerGivenContextMenuMock, scenarioSimulationGridPanelClickHandler.headerGivenContextMenu);
     }
 
     @Test
     public void setHeaderExpectedContextMenu() {
-        scenarioSimulationGridPanelClickHandler.setHeaderExpectedContextMenu(mockHeaderExpectedContextMenu);
-        assertEquals(mockHeaderExpectedContextMenu, scenarioSimulationGridPanelClickHandler.headerExpectedContextMenu);
+        scenarioSimulationGridPanelClickHandler.setHeaderExpectedContextMenu(headerExpectContextMenuMock);
+        assertEquals(headerExpectContextMenuMock, scenarioSimulationGridPanelClickHandler.headerExpectedContextMenu);
```
Comment (Rikkola) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/test/java/org/drools/workbench/screens/scenariosimulation/client/handlers/ScenarioSimulationGridPanelClickHandlerTest.java]: I like having the mock at the end. Easier to read.

Comment (gitgabrio) [reply drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/test/java/org/drools/workbench/screens/scenariosimulation/client/handlers/ScenarioSimulationGridPanelClickHandlerTest.java]: Yup. I'm going to apply this name convention on all mocked objects

**Comment by gitgabrio** [comment]
@manstis 
Hullo! Could you please make the magic merge (squash commits) -? THanks!

**Pull request merged**